setup.py

```diff
@@ -14,7 +14,7 @@
         'console_scripts': ['patsy=patsy.__main__:main']
         },
     install_requires=[i.strip() for i in open("requirements.txt").readlines()],
-    python_requires='>=3.7',
+    python_requires='>=3.10',
```

Comment:
Unless you are using features specific to a version of Python later than 3.7, it is okay to keep this as `>=3.7`, since it is just the specification of the *minimum* version necessary to run.